testing/python/language/test_tilelang_language_vectorize.py

```
@@ -0,0 +1,67 @@
+from tilelang import tvm as tvm
+import tilelang.testing
+import tilelang as tl
+import tilelang.language as T
+
+
+def vectorize_test(N, M, stride_A, stride_B):
+    assert N % 128 == 0 and M % 128 == 0
+
+    @T.prim_func
+    def main(
+        A: T.StridedTensor[(N, M), (1, stride_A), "float32"],
```

Ruff (0.14.0)

Undefined name `float32`

(F821)